src/bentoml/_internal/context.py

```diff
@@ -72,21 +73,38 @@ def mutablecopy(self) -> Metadata:
         """
 
 
-@attr.define
 class InferenceApiContext:
```

Comment:
This isn't an API context anymore, right? We should probably rename it to reflect that.